java/struts2/pom.xml

```diff
@@ -73,7 +73,6 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-assembly-plugin</artifactId>
-                <version>3.2.0</version>
```

Comment:
I didn't test that and then it will use version related to Maven used to build the project.